django/core/paginator.py

```diff
@@ -96,19 +95,35 @@ def _check_object_list_is_ordered(self):
         """
         Warn if self.object_list is unordered (typically a QuerySet).
         """
-        if hasattr(self.object_list, 'ordered') and not self.object_list.ordered:
+        if hasattr(self.object_list,
+                   'ordered') and not self.object_list.ordered:
             warnings.warn(
                 'Pagination may yield inconsistent results with an unordered '
                 'object_list: {!r}'.format(self.object_list),
                 UnorderedObjectListWarning
             )
 
+    def page_force_into_range(self, number):
```

Comment:
I'm not entirely sure about this name. I don't have any better suggestions right now though.